src/_assets/css/_dash.scss

```diff
@@ -328,12 +328,24 @@ $dash-dartpad-border: #293542;
 .banner {
   background-color: $dash_callout;
 
-  .banner__text {
+  .bw_banner__text {
     color: white;
+
+    a {
+      color: white;
+      text-decoration: underline;
+      font-weight: 700;
+    }
```

Comment:
Great to see this banner!

Btw, defining a new class **breaks the formatting** a bit (notice that there's a bottom margin on the banner, the font isn't quite right, etc):

> ![image](https://user-images.githubusercontent.com/4140793/87186823-22b1ba80-c2ba-11ea-9bb2-246a27e39941.png)

You'll want to keep the `.banner__text` name and instead, say, introduce a class (like `a.bw`) below `.banner__text` so that it looks like this:

> ![image](https://user-images.githubusercontent.com/4140793/87187257-e03cad80-c2ba-11ea-8bfd-8f2403c95911.png)

Reply:
Or do something similar to the Flutter website: define a `.banner__text--bw` style to use as _an additional_ class on the banner div paragraph.

Reply:
Thank you! I saw the problem but assumed I'd just never noticed it before.